### PR TITLE
QA_AUTO cleanup

### PR DESCRIPTION
**Pull request by mimi1vx**
http://argus.suse.cz/tests/1445
http://argus.suse.cz/tests/1446

### CODE REVIEW

File: tests/qa_automation/qa_run.pm [hunk 1]
```diff
@@ -109,20 +108,17 @@ sub qa_script_output {
...
     # Set a high timeout value for wait_serial
     # so that it will wait until test run finished or
     # MAX_JOB_TIME(can be set on openQA webui) reached
     my $ret = wait_serial($pattern, 180 * 60);
-    if ($ret) {
-        return 1;
-    }
-    return 0;
+    return 1 if $ret;
+    return undef;
```
Comment (coolo) [inline tests/qa_automation/qa_run.pm]: 'return undef' is the same as 'return;' but longer. but in this very case I would suggest 'return $ret ? 1 : 0;'

File: tests/qa_automation/qa_run.pm [hunk 2]
```diff
@@ -109,20 +108,16 @@ sub qa_script_output {
 
 # Check whether DONE file exists every $interval secs in the background
 sub wait_testrun {
-    my ($self, $interval) = @_;
```
Comment (coolo) [inline tests/qa_automation/qa_run.pm]: stick to the unpack variant - you can still use $interval //= 30; to set a default

File: tests/qa_automation/qa_run.pm [hunk 3]
```diff
@@ -9,8 +9,8 @@
 # without any warranty.
 #
 package qa_run;
-# G-Summary: remove code duplication by sharing the code in a base class
-# G-Maintainer: Stephan Kulow <coolo@suse.de>
+# Summary:  base class for qa_automation tests in openQA
+# Maintainer: Stephan Kulow <coolo@suse.de>
```
Comment (coolo) [inline tests/qa_automation/qa_run.pm]: meanwhile I guess it's rather nathan or sero

File: tests/qa_automation/qa_run.pm [hunk 4]
```diff
@@ -50,12 +50,12 @@ sub create_qaset_config {
...
-    my $self = shift;
-    my $qa_server_repo = get_var('QA_SERVER_REPO', '');
+    my $self           = shift;
+    my $qa_server_repo = get_var('QA_SERVER_REPO');
+    pkcon_quit;
     if ($qa_server_repo) {
         # Remove all existing repos and add QA_SERVER_REPO
-        my $rm_repos = "declare -i n=`zypper repos | wc -l`-2; for ((i=0; i<\$n; i++)); do zypper rr 1; done; unset n; unset i";
-        assert_script_run($rm_repos, 300);
+        script_run('for i in {1..$(zypper lr| tail -n+3 |wc -l)}; do echo $i; done; unset i', 300);
```
Comment (dzedro) [inline tests/qa_automation/qa_run.pm]: echo will not remove repos

Comment (mimi1vx) [inline tests/qa_automation/qa_run.pm]: thx, typo

File: tests/qa_automation/qa_run.pm [hunk 5]
```diff
@@ -68,35 +68,34 @@ sub prepare_repos {
...
 
 # Create qaset/config file, reset qaset, and start testrun
 sub start_testrun {
     my $self = shift;
     $self->create_qaset_config();
-    assert_script_run "/usr/share/qa/qaset/qaset reset";
+    assert_script_run("/usr/share/qa/qaset/qaset reset");
     my $testsuite = $self->test_suite();
-    assert_script_run "/usr/share/qa/qaset/run/$testsuite-run.openqa";
+    assert_script_run("/usr/share/qa/qaset/run/$testsuite-run.openqa");
```
Comment (dzedro) [inline tests/qa_automation/qa_run.pm]: Is this really needed ? I don't mind :)

Comment (mimi1vx) [inline tests/qa_automation/qa_run.pm]: same style in whole file ... 

I prefer with () :) I can accept without .. but I really don't like mixed

Comment (dzedro) [inline tests/qa_automation/qa_run.pm]: fine

**Pull request merged**